### DISCUSSION TITLE
Web/Web/ Error ?

### DISCUSSION THREAD

**Discussion by PatH007** [LibreBooking/librebooking#411] [Q&A] [answered]
Hi there.
I just start to install librebooking on my server via FTP.
I tried to adapt config.php and tried https://mywebsite.fr/librebooking/Web, but it automatically goes back to https://mywebsite.fr/librebooking/Web/Web with an 404 error...
No idea?
Is there any relationship with session.autostart ?
thanks for any help.

**Comment by vd3030** [comment]
same here. not working. just go back to 2.8.5...

**Reply by vd3030** [reply]
I just checked the config.php, and comment out the 'script url', it works. I think they are fixing the bugs, but still many...

**Comment by apezzi75** [comment, marked as answer]
you have to modify the .htacces file :

RewriteCond %{REQUEST_URI} !^/librebooking/Web/
RewriteRule ^(.*)$ /librebooking/Web/$1 [R]
